File: app/src/main/java/com/itba/runningMate/pastruns/PastRunsView.java
package com.itba.runningMate.pastruns;

import com.itba.runningMate.domain.Run;

import java.util.List;

public interface PastRunsView {

    void updatePastRuns(List<Run> list);

    void showNoPastRunsMessage();

    void hideNoPastRunsMessage();

    void launchRunDetails(long id);
}

File: app/src/main/java/com/itba/runningMate/pastruns/PastRunsView.kt
package com.itba.runningMate.pastruns

import com.itba.runningMate.domain.Run

interface PastRunsView {

    fun updatePastRuns(list: List<Run>)

    fun showNoPastRunsMessage()

    fun hideNoPastRunsMessage()

    fun launchRunDetails(id: Long)

    fun showDeleteError()

}